Yandex/Yandex/PassportPasswordPage.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System.Threading;
+
+namespace Yandex
+{
+    public class PassportPasswordPage : BasePage
+    {        
+        const string PasswordInputId = "passp-field-passwd";
+        const string SignInButtonId = "passp:sign-in";
+
+        Dictionary<string, IWebElement> _elementCollection;
+
+        public PassportPasswordPage(IWebDriver driver) : base(driver)
+        {
+            var passwordInput = GetElementById(PasswordInputId);
```

Comment:
I recommend you to find elements within methods where they are need to be used. As for your class, find elements within EnterPassword method.